sessionctx/variable/tidb_vars.go

```diff
@@ -274,6 +274,9 @@ const (
 
 	// TiDBUseAlloc indicates whether the last statement used chunk alloc
 	TiDBUseAlloc = "last_sql_use_alloc"
+
+	// TiDBExplicitRequestSourceType indicates the source of the request, which can be ["lightning", "br", "dumping", "tispark", "background"].
```

Comment:
Should we expose `"lightning", "br", "dumping", "tispark", "background"` as const? Maybe we need to use them both in TiDB and client-go.

Reply:
my should defined in client-go.